core/scripts/cli/voting/getResolvedVotesByRoundId.js

```diff
@@ -0,0 +1,76 @@
+const style = require("../textStyle");
+
+// Given a list of revealed votes for a user, return
```

Comment:
Awesome comment here. Would you mind adding a short (doesn't have to be this detailed) comment above all of the methods you export in other files?

Reply:
added comments to every single method in `cli/`